tests/decorators/decorators.go

```diff
@@ -4,7 +4,8 @@ import . "github.com/onsi/ginkgo/v2"
 
 var (
 	//
-	Quarantine = []interface{}{Label("QUARANTINE")}
+	Quarantine     = []interface{}{Label("QUARANTINE")}
+	Periodics_only = []interface{}{Label("PERIODICS_ONLY")}
```

Comment:
nit: I'd rather use CamelCase for the name.
```suggestion
	PeriodicsOnly = []interface{}{Label("PERIODICS_ONLY")}
```

Reply:
Why not `Periodic`?

Reply:
+1 for just `Periodic`